client-js/app/scripts/app.ts

```diff
@@ -23,10 +25,11 @@ window.addEventListener('WebComponentsReady', function() {
     page(e.detail.path);
   });
 
-  var socket = new registry.JsonRpcSocket('ws://localhost:9000/api/socket2');
+  var socket = new clientRpc.JsonRpcSocket('ws://localhost:9000/api/socket2');
+  socket.prefix = "org.labrad."
   var reg = new registry.RegistryServiceJsonRpc(socket);
+  var echo = new echoServer.EchoServiceJsonRpc(socket);
```

Comment:
When I made the prefix a property of the socket I was thinking it would be cleaner if different sockets were opened to make calls with different prefixes, so all calls made to the manager would have the ```org.labrad``` prefix. If I were going to make calls to a different service I might as well open a new socket.